src/harness/sas.py

```diff
@@ -304,9 +304,124 @@ def InjectCpiUser(self, request):
     _RequestPost('https://%s/admin/injectdata/cpi_user' % self._base_url,
                  request, self._tls_config)
 
+  def TriggerCreateActivityDump(self, request):
+    _RequestPost('https://%s/admin/trigger/create_full_activity_dump' % self._base_url,
+                 request, self._tls_config)
+
+  def InjectPeerSas(self,request):
+    _RequestPost('https://%s/admin/injectdata/peer_sas' % self._base_url,
+                 request, self._tls_config)
+
   def _GetDefaultAdminSSLCertPath(self):
     return os.path.join('certs', 'admin_client.cert')
 
   def _GetDefaultAdminSSLKeyPath(self):
     return os.path.join('certs', 'admin_client.key')
 
+def GetServer():
+  config_parser = ConfigParser.RawConfigParser()
+  config_parser.read(['sas.cfg'])
+  server_base_url = config_parser.get('HttpServer', 'BaseUrl')
```

Comment:
We will need to support multiple SAS test harnesses. Maybe it's easier to let these be part of the individual test config?

Reply:
Already moved this GetServer() function wrapped into startServer() function in util.py file